instrumentation/ktor/ktor-2.0/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v2_0/client/KtorClientTracingBuilder.kt

```diff
@@ -37,23 +40,104 @@ class KtorClientTracingBuilder {
     httpAttributesExtractorBuilder.setCapturedRequestHeaders(headers)
   }
 
+  fun capturedRequestHeaders(vararg headers: String) {
+    capturedRequestHeaders(headers.asIterable())
+  }
+
+  fun capturedRequestHeaders(headers: Iterable<String>) {
+    setCapturedRequestHeaders(headers.toList())
+  }
+
   fun setCapturedResponseHeaders(vararg headers: String) = setCapturedResponseHeaders(headers.asList())
 
   fun setCapturedResponseHeaders(headers: List<String>) {
     httpAttributesExtractorBuilder.setCapturedResponseHeaders(headers)
   }
```

Comment:
@marychatte would it make sense to deprecate the `set*()` methods?

Reply:
Yes, thanks. I will do it